docs/1___overview.adoc

```diff
@@ -172,12 +176,12 @@ Else: icon:times[]
 
 === Properties and Guiding Ideas
 
-In this section, properties are listed and some principles are defined that guided the low-level design of the FMI.
-This shall increase self consistency of the FMI functions.
-The listed issues are sorted, starting from high-level properties to low-level implementation issues.
+In this section, properties are listed and some principles are defined that guide the design of the FMI API and XML schema itself (not the content of the FMUs).
+Following these principles shall help increase self consistency of the FMI API functions.
```

Comment:
I am not sure what you mean by "self consistency of the FMI API functions".

Reply:
I have an idea who wrote this a long time ago :)
Well, even outside psychology, "self consistency" requires "likeness of approach within the group".
In essense, this covers trivially naming conventions, argument ordering, array vs. pointer usage, but also more complex behavioral concepts like error handling, return value use, side-effect conventions and the like.
I am quite happy to have this vague here as it should capture whatever we mean in terms of similarity between the API functions or even the XML elements and their structure.
If you have a better way to describe this to mean "as-similar-as-possible-with-respect-to-natural-differences" - I am open for suggestions.